webapp/src/Command/ImportEventFeedCommand.php

```diff
@@ -54,94 +58,29 @@ class ImportEventFeedCommand extends Command
     const STATUS_OK = 0;
     const STATUS_ERROR = 1;
 
-    /**
-     * @var EntityManagerInterface
-     */
-    protected $em;
-
-    /**
-     * @var DOMJudgeService
-     */
-    protected $dj;
-
-    /**
-     * @var ConfigurationService
-     */
-    protected $config;
-
-    /**
-     * @var EventLogService
-     */
-    protected $eventLogService;
-
-    /**
-     * @var ScoreboardService
-     */
-    protected $scoreboardService;
-
-    /**
-     * @var SubmissionService
-     */
-    protected $submissionService;
-
-    /**
-     * @var TokenStorageInterface
-     */
-    protected $tokenStorage;
-
-    /**
-     * @var bool
-     */
-    protected $debug;
-
-    /**
-     * @var string
-     */
-    protected $domjudgeVersion;
-
-    /**
-     * @var LoggerInterface
-     */
-    protected $logger;
-
-    /** @var HttpClientInterface|null */
-    protected $client;
-
-    /** @var string */
-    protected $configKey;
-
-    /** @var int|string */
-    protected $contestIdFromConfig;
-
-    /** @var int */
-    protected $contestId;
-
-    /** @var string|null */
-    protected $feedUrl;
-
-    /** @var string|null */
-    protected $feedFile;
-
-    /** @var string|null */
-    protected $basePath;
-
-    /** @var bool */
-    protected $allowImportAsPrimary;
-
-    /** @var bool */
-    protected $allowExternalIdMismatch;
-
-    /** @var string|null */
-    protected $sinceEventId = null;
-
-    /** @var bool */
-    protected $shouldStop = false;
-
-    /** @var string|null */
-    protected $lastEventId = null;
-
-    /** @var array */
-    protected $verdicts = [];
+    protected EntityManagerInterface $em;
+    protected DOMJudgeService $dj;
+    protected ConfigurationService $config;
+    protected EventLogService $eventLogService;
+    protected ScoreboardService $scoreboardService;
+    protected SubmissionService $submissionService;
+    protected TokenStorageInterface $tokenStorage;
```

Comment:
consider adding new lines where it makes sense from a grouping perspective, e.g. after all the "injected" variables

or below to group the feed params

Reply:
I added some lines. Might be more classes where we want it. Maybe we just merge at some point and people add when they want it?